### ISSUE TITLE
Let's show multiple AI accelerators supported by agents(resource group) in resource monitor per AI accelerator

### ISSUE THREAD

**Issue by lizable** [lablup/backend.ai-webui#1898]
### Main idea

For now, we've been only providing "NVIDIA GPU" based AI Accelerator when creating session and listing the allocated amount of resources in resource panel. As time passes by, we are now supporting multiple AI Accelerators on agents. which means that this triggers we need to provide users to choose one of AI accelerators on demand, not just showing NVIDIA-oriented GPU. This issue needs to be handled in every UI that is related to session creation/listing amount of resources.

I just confirmed that we can list up like this:
<img width="374" alt="Resource Monitor" src="https://github.com/lablup/backend.ai-webui/assets/46954439/f610878d-5ad1-434a-95bb-158d4cb9fa17">

but layout breaks when the horizontal length of the screen changes.
<img width="544" alt="MicrosoftTeams-image (9)" src="https://github.com/lablup/backend.ai-webui/assets/46954439/7a13d753-d25f-4b28-9e07-994db97af648">

In short, We need to fix the broken layout when the agent supports multiple AI accelerators (more than one).

**Comment by lizable** [comment]
Closing this issue since it's been resolved.
<img width="890" alt="Screenshot 2023-11-23 at 6 54 08 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/fac9fb93-5124-489a-8a30-72a39d699e51">
